tests/Filesystem/FilesystemTest.php

```diff
@@ -324,4 +324,40 @@ public function testSharedGet()
         $this->assertTrue($result === 1);
         @unlink(__DIR__.'/file.txt');
     }
+
+    public function testRequireOnceRequiresFileProperly()
```

Comment:
You should call requireOnce more than one to check it only requires the file once.